hudi-common/src/main/avro/HoodieMetadata.avsc

```diff
@@ -358,6 +358,45 @@
                 }
             ],
             "default" : null
+        },
+        {
+            "name": "recordLevelIndexMetadata",
+            "doc": "Contains information about record keys and their location in the dataset",
+            "type": [
+                "null",
+                {
+                    "type": "record",
+                    "name": "HoodieMetadataRecordLevelIndex",
+                    "fields": [
+                        {
+                            "name": "partition",
+                            "type": "string",
+                            "doc": "Partition which contains the record",
+                            "avro.java.string": "String"
+                        },
+                        {
+                            "name": "fileId",
+                            "type": "string",
+                            "doc": "FileId of file group"
+                        },
+                        {
+                            "name": "rowGroupIndex",
```

Comment:
may I know whats the purpose of row group index ?

Reply:
Thanks for review.
This is a parameter reserved for parquet partial updates
https://github.com/apache/hudi/pull/6612